web/styles/zulip.css

```diff
@@ -2522,14 +2522,30 @@ select.invite-as {
     }
 }
 
-#invite-method-choice {
-    margin-top: 2px;
+.invite_type_radio_section {
+    margin: 2px 0 2px 5px;
+    padding: 2px 0;
+
+    & label.display-settings-radio-choice-label {
+        display: inline;
+        margin: 5px;
+
+        & input[type="radio"] {
+            top: 0;
+        }
+    }
+
+    .control-label-disabled label {
+        cursor: not-allowed;
+
+        & input[type="radio"] {
+            pointer-events: none;
```

Comment:
Is this needed? I cannot see any change after I remove this, but maybe I am missing something.

Reply:
[simplescreenrecorder-2023-05-30_21.44.38.webm](https://github.com/zulip/zulip/assets/99073049/55b3cc9e-ac7e-4b3f-be0d-775aafa715d3)

Reply:
I was facing this issue of tooltip not visible without `pointer-events`.